SECURITY.md

```diff
@@ -1,9 +1,32 @@
 # Security Policy
 
-## Supported Versions
+## Reporting a Vulnerability
 
-See our [Version Compatibility Documentation](https://docs.konghq.com/kubernetes-ingress-controller/latest/references/version-compatibility/#kubernetes).
+At Kong, we take security issues very seriously. If you believe you have found a security vulnerability in our project, we encourage you to disclose it responsibly. Please report any potential security vulnerabilities to us by sending an email to [vulnerability@konghq.com](mailto:vulnerability@konghq.com).
 
-## Reporting a Vulnerability
+## How to Report
+
+1. **Do not publicly disclose the vulnerability**: Please do not create a GitHub issue or post the vulnerability on public forums. Instead, contact us directly at [vulnerability@konghq.com](mailto:vulnerability@konghq.com).
```

Comment:
What about reporting through GH

<img width="1330" alt="image" src="https://github.com/user-attachments/assets/31161181-e866-4a03-8e4c-dec86a062661" />

it's confidential

<img width="924" alt="image" src="https://github.com/user-attachments/assets/48181971-6139-4470-b3c0-63de8c0e5548" />

in my opinion, this is the most idiomatic way for OSS project hosted on GH

Reply:
Ok